src/components/views/project/ProjectGIVbackToast.tsx

```diff
@@ -164,13 +170,27 @@ const ProjectGIVbackToast = () => {
 						/>
 					</ButtonWrapper>
 				)}
+				{eligibleForVerification && (
+					<ButtonWrapper>
+						<OutlineButton
+							onClick={() => setShowVerificationModal(true)}
+							label='Verify Project'
+							icon={<IconVerifiedBadge16 />}
+						/>
+					</ButtonWrapper>
+				)}
 			</Wrapper>
 			{showBoost && (
 				<BoostModal
 					projectId={projectData?.id!}
 					setShowModal={setShowBoost}
 				/>
 			)}
+			{showVerificationModal && (
+				<VerificationModal
+					onClose={() => setShowVerificationModal(false)}
+				/>
```

Comment:
to avoid multiple re-renders pls use useCallback() , exmples here https://react.dev/reference/react/useCallback#usage